crates/turbopack-image/src/process.rs

```diff
@@ -109,32 +172,45 @@ fn compute_blur_data(
                     blur_height,
                     small_image.color(),
                 )?;
-            format!(
-                "data:image/webp;base64,{}",
-                Base64Display::new(&buf, &STANDARD)
-            )
+            (buf, Mime::from_str("image/webp")?)
         }
         #[cfg(feature = "avif")]
         ImageFormat::Avif => {
             use image::codecs::avif::AvifEncoder;
-            AvifEncoder::new_with_speed_quality(&mut buf, 6, options.quality).write_image(
-                small_image.as_bytes(),
-                blur_width,
-                blur_height,
-                small_image.color(),
+            AvifEncoder::new_with_speed_quality(&mut buf, 6, quality - 15).write_image(
```

Comment:
These should go into constants that explain how we came up with these numbers. Why `- 15` here?

Reply:
removed it... I copied that from next.js but there seem to be no reason for that...